en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md

```diff
@@ -732,4 +732,20 @@ Run the following CTL command to change the default location of the export direc
         - Required :   
             `--export-directory`: Path to directory where APIs should be saved.   
             Default : `/home/.wso2apictl/exported`
+            
+            
+## Import SSL Certificate for Secure HTTP Communication with API Manager
+
+Different environments of API Manager can have different SSL certificates for secure HTTP communications. The default
+certificate of WSO2 API Manager is a self signed certificate and in production environments it is advised to use a
```

Comment:
"certificate of WSO2 API Manager is a self signed certificate and in production environments it is advised to use" shall we change this as follows
"certificate of WSO2 API Manager is a **self-signed** certificate and in production **environments, it is** advised to use "
```suggestion
certificate of WSO2 API Manager is a self-signed certificate and in production environments, it is advised to use a
```

Reply:
Fixed